js/demo/lazy-loading.js

```diff
@@ -26,14 +26,15 @@ var EffecktLazyLoading = {
 
   bindUIActions: function() {
 
-    var self = this;
+    var self = this,
+      windowObj = $(window);
```

Comment:
Since $(window) was being used twice, I thought of creating a variable and assigning $(window) to that. Will it be OK if I use $(window) twice here?